models/music_album.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(publish_date, archived, on_spotify: false)
+    super(publish_date, archived: archived)
+    @on_spotify = on_spotify
```

Comment:
- I don't think you need to pass archived in the child class cause that is getting taken care of by the `can_be_archived?` overridden method.
> `publish_date` is a required parameter in `Item` class so only that needs to be passed through `super` and that will > be enough.

- **[Optional]** You can pass a `public` parameter `name` or `title` in the `MusicAlbum` class. This will help you identify the `MusicAlbum` with the name in the list when you will implement the list in the upcoming task